android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/LogHandlerWithMapping.kt

```diff
@@ -0,0 +1,30 @@
+package com.revenuecat.purchases.hybridcommon.mappers
+
+class LogHandlerWithMapping(
+    private val callback: (logData: Map<String, String>) -> Unit
+) : com.revenuecat.purchases.LogHandler {
+    override fun d(tag: String, msg: String) {
+        invokeCallback("debug", msg)
+    }
+
+    override fun e(tag: String, msg: String, throwable: Throwable?) {
+        val message = throwable?.let { "$msg. Throwable: $it" } ?: msg
+        invokeCallback("error", message)
+    }
+
+    override fun i(tag: String, msg: String) {
+        invokeCallback("info", msg)
+    }
+
+    override fun v(tag: String, msg: String) {
+        invokeCallback("verbose", msg)
+    }
+
+    override fun w(tag: String, msg: String) {
```

Comment:
We are ignoring `tag` in all this... I wonder if we should embed it into the message...

Reply:
I didn't because iOS doesn't have one, so it would be weird Android logs have a [PURCHASES] in front. But if you think it's worth it to add it I will

Reply:
Hmm not a big deal either way I would say. I think it's fine to leave as is.